enarx-srv-traits/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "enarx-srv-traits"
+version = "0.1.0"
+authors = ["Nathaniel McCallum <npmccallum@redhat.com>"]
+edition = "2018"
+
```

Comment:
We may need a license field here, otherwise if #162 is merged first, we might get barked at.

Reply:
I fixed this in the forthcoming push.